pkg/metrics/metrics.go

```diff
@@ -159,6 +159,108 @@ var (
 		Name:      "policy_l7_received_total",
 		Help:      "Number of total L7 received requests/responses",
 	})
+
+	// Status
+
+	// ControllersFailing is a count of failing controllers
+	ControllersFailing = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: Namespace,
+		Name:      "controllers_failing_total",
```

Comment:
Gauges shouldn't have `_total` suffix; try running this and use `promtool check metrics` for similar feedback.